apps/modernization-ui/src/date/asAgeDisplay.ts

```diff
@@ -0,0 +1,26 @@
+import { differenceInDays, differenceInMonths, differenceInYears } from 'date-fns';
+
+const asAgeDisplay = (dateOfBirth: string | Date | null | undefined, from: Date = new Date()): string | undefined => {
```

Comment:
suggestion (non-blocking): There are some other functions( [displayName](https://github.com/CDCgov/NEDSS-Modernization/blob/main/apps/modernization-ui/src/name/displayName.ts) and [displayAddress](https://github.com/CDCgov/NEDSS-Modernization/blob/main/apps/modernization-ui/src/address/display/displayAddress.ts)) for displaying things that are named `displayX` should we keep that pattern?  

nit: I may have been corrupted by too much functional programming but if you move the parameters around and add some currying it becomes a little more expressive.

`const asAgeDisplay = (from: Date = new Date()) = (dateOfBirth: Maybe<string | Date>) => {...}`. you could then write functions from this like `const asAgeFromToday = asAgeDisplay();`

Reply:
Will add this as part of Mortality PR